nrrd/reader.py

```diff
@@ -250,7 +255,12 @@ def read_header(file, custom_field_map=None):
 
         # Check if the field has been added already
         if field in header.keys():
-            raise NRRDError('Duplicate header field: %s' % repr(field))
+            dup_message = "Duplicate header field: '%s'" % str(field)
+
+            if not _NRRD_ALLOW_DUPLICATE_FIELD:
+                raise NRRDError(dup_message)
+
+            warnings.warn(dup_message)
```

Comment:
So with a duplicated field, the old field will be overwritten. Would it be better if we added an integer to the end of the field. Something like <field>2 so that it doesn't conflict but both of the values are saved?

Reply:
I think it's fine to overwrite. If the values were duplicated then there would probably need to be a hack in the save path to remove the second key while saving, which would just be confusing.